### DISCUSSION TITLE
Export to single file possible?

### DISCUSSION THREAD

**Discussion by arough007** [audiamus/AaxAudioConverter#101] [Q&A] [answered]
Hi,
is it possible to export all selected .aax files to a single m4a/m4b?
If I have a downloaded audiobook with more than one part, the tool creates one m4a/m4b for every part right now.

Either I'm blind or there is no such functionality.
Anyway, thanks in advance for the help.

**Comment by audiamus** [comment]
No, there is no such functionality, I'm afraid. 

As soon as a book consists of more than one track, be it because it has parts, or because it is split into chapters and more, AAX Audio Converter automatically will create a playlist. Many media players can read the playlist file format. But I am aware some do not. For those cases the output file names will be in correct alphabetic play order either by default or can be made so with some of the file and folder naming options. 

For a book with multiple parts, AAX Audio Converter will remove the continuation hint at the end of an intermediate part and also the repeated intro at the beginning of the next part. So, if you listen to your book either via the playlist or with a player that works through the files in the folder alphabetically, you shouldn't notice the change of file.

If you absolutely need it as a single file book, you can download it as such from Audible, but you know that, of course. (There may be some exceptions for very thick volumes.)

**Comment by arough007** [comment]
That's a bummer.
I have 13 titles in my library that are multiple parts (up to seven parts), even when set to single parts in Audible.

I decode my audiobooks with Leaax_Decode into seperate files and bunch them all together in one m4b file via fre:ac.
I had hoped to have found a program to do all this in one single step.
I use single files because it's easier to organize IMO and because my audiobook player is doing weird stuff with audiobooks with multiple parts.

Is there any possibilty you could add this functionality?

**Comment by audiamus** [comment, marked as answer]
It's rather unlikely, I'd say. The concept was to spilt files, not combine them, and the entire workflow is built around that. It would need major changes.

And currently, my focus is on AAX Audio Converter 2 and a built-in direct download function.

**Comment by arough007** [comment]
No problem.
Thanks for your replies ;)

I found out that if I just add the m3u my app is working better at registring it's a single audiobook and not multiple different ones with oddly similiar titles =)
Smart Audio Book Player on Android worked without m3u files to guide it, so I just assumed it was the same with my app on the iPhone.
So just like always: I was the dumb one^^
